src/main/java/com/javadeobfuscator/deobfuscator/transformers/special/ParamorphismTransformer.java

```diff
@@ -359,6 +346,37 @@ else if(replacements.get(caller) instanceof FieldInsnNode)
 				}
 			}
 		}
+		classNodes().stream().parallel().forEach(cn -> {
```

Comment:
Hmmmm, is there a justification for using a parallel stream?

Reply:
We dont need to care the order when decrypt the hide access.
i think parallel stream makes it faster.

Reply:
Have you checked if it's actually faster?  I would imagine that the performance should roughly be about equal given that 1. there are typically not enough methods to warrant parallel execution and 2. `System.out.println(String)` is synchronized in OpenJDK JVMs.

Reply:
Sorry I forget System.out